In [13]:
load('../VeeringGeneralFunctions.sage')
sig = 'ivLLQQccdgffhghhvvaaaaavv_01111220'

Recall the formula for the Thurston norm:
$$ \| \alpha \|_T = 2\text{genus}(\alpha) + \#\text{punctures}(\alpha) - 2 $$
giving that
$$ \text{genus}(\alpha) = \frac{1}{2}\left(\| \alpha \|_T - \#\text{punctures}(\alpha) + 2\right) $$

In [14]:
cone = get_fibered_cone(sig)
cone.Hilbert_basis()

N(0, 0,  1),
N(1, 0, -1),
N(1, 1,  2),
N(2, 1,  0)
in 3-d lattice N

In [15]:
get_fibered_cone_plot(sig)

Graphics3d Object

In [18]:
get_spec_with_subs(sig, (2,1.5,3))

-x^10.0000000000000 + x^9.50000000000000 + x^9.00000000000000 + 2*x^8.00000000000000 + x^7 + x^6.50000000000000 - x^6.00000000000000

In [19]:
get_Thurston_norm(sig)

2*a0 - 2*a1 + a2

In [20]:
get_min_norm_dila_log_mathematica_approx(sig)

3.6538111277006253

In [21]:
maps = get_LES_boundary_matrices(sig)
maps

[
[-3  6 -2]  [-1  3  0]  [ 0  0 -1]
[ 0  1  0], [ 1  0  0], [ 0 -2  0]
]

In [22]:
maps[0]*vector([2,1,2*var('g')-1]), maps[1]*vector([2,1,2*var('g')-1]), maps[2]*vector([2,1,2*var('g')-1])

((-4*g + 2, 1), (1, 2), (-2*g + 1, -2))

Consider the classes $(2,1,2g-1)$ for $g$ ranging from $1$ to infinity. Using the above information,
$$ \| (2,1,2g-1) \|_T = 2*2 - 2*1 + 2g-1 = 2g+1 $$
and by the previous block, we have
$$\#\text{punctures}(2,1,2g-1) = \gcd(-4g + 2, 1) + \gcd(1, 2) + \gcd(-2g + 1, -2) = 2 + \gcd(-2g + 1, -2) = 3$$
where $\gcd(-2g+1, -2) = 1$ because the first term is always odd.

Thus
\begin{align*}
\text{genus}(2,1,2g-1) &= \frac{1}{2}\left( \color{red}{\| (2,1,2g-1) \|_T} \color{blue}{- \#\text{punctures}(2,1,2g-1)} \color{green}{+ 2}\right) \\
&= \frac{1}{2}\left(\color{red}{2g+1} \color{blue}{- 3} \color{green}{+ 2}\right) \\
&= g
\end{align*}

What is more, one can see again by the previous code block that for all $g$, the slope on the first (using zero-indexing) boundary component is $2/1$; in other words, it doesn't depend on $g$ so they all lie on the same slope of the first boundary component. That is, all these classes lie in the Dehn filling of the first boundary component by slope $2/1$.

To see that these classes actually represent pseudo-Anosov monodromies after Dehn filling, we should observe that they have no one-prongs. To do this, we will use a version of the VeeringGeneralFunction get_num_prongs(sig, point), but we have to be careful because this function won't let us use the variable $g$ that we have been using. So we work through that function piece by piece being careful with the variable.

In [23]:
point = (2,1,2*var('g') - 1)
substitution_dict = {get_hom_vars(sig)[i]:point[i] for i in range(len(point))} # substitute the given point for our variable place holders.
two_chain_with_free_vars = get_2_chain_from_point_in_homology(sig).subs(substitution_dict)

In [24]:
two_chain_with_free_vars

(r53, 2*g - r51 - r52 - r55 - r56 - 1, r52, -2*g + r52 + r56 + 2, r51, 2*g - r52 - r53 - r54 - r56, r56, -r51 - r55 - r56 + 1, -r50 + r51 + r56 + 1, -2*g + r51 + r52 + r53 + r54 + r55 + r56, r55, r54, -r50 + r51 + r52 + r53 + r56 - 1, 2*g - r51 - r52 - r53 - r56, 2*g + r50 - r51 - r52 - r53 - r54 - r56, r50)

In [25]:
no_free_vars = [] # two_chain_with_vars still contains the free variables. We want to set those all to 0. But NOT g!
for term in two_chain_with_free_vars:
    no_free_vars.append(term.subs({v:0 for v in term.variables() if v != var('g')})) # don't replace the variable g with 0 here!

In [26]:
no_free_vars

[0, 2*g - 1, 0, -2*g + 2, 0, 2*g, 0, 1, 1, -2*g, 0, 0, -1, 2*g, 2*g, 0]

In [27]:
no_free_vars = vector(no_free_vars)

In [28]:
total_prongs_each_boundary_comp = get_prongs_matrix(sig)*no_free_vars

In [29]:
total_prongs_each_boundary_comp

(4*g - 2, 2, 2)

The above tells us that all punctures coming from the zeroth boundary component split $4g-2$ prongs, while those coming from the first and second boundary components split $2$ prongs respectively. As every boundary component contributes only one puncture (seen above), there are never any punctures with $1$-prongs.

The last thing to observe is that if we Dehn fill the first boundary component along the slope $2/1$, we get a minimizer which is smaller than the minimizer of the unfilled manifold: $3.6538111277006253$.
For this, notice that the point (2,1,2) lies on this slope and the Thurston norm of the Dehn filling of this point is the same as the Thurston norm of the unfilled point, but after filling there is one less puncture (since the first boundary component contributes one puncture).

In [30]:
(2*2 - 2*1 + 2 - 1)*math.log(get_dila_mathematica(sig, (2,1,2))) # minus one to account for one less puncture

2.887270950357621

As this is smaller than the normalized dilatation of the unfilled manifold and necessarily larger than the minimimum normalized dilatation of the Dehn filled manifold, we conclude that the Dehn filled manifold's minimizer is strictly smaller than the unfilled minimizer.

Does this allow us to conclude that the smallest dilatation examples are on finitely many slopes? Isn't it possible to have a class with the same genus and a larger normalized dilatation, but a smaller dilatation? According to KKT page 3583 (proof of Theorem 1.4(1)) I think this is all we need.

Also should observe that these classes are actually in the interior of the fibered cone!